stylistic.js

```diff
@@ -49,9 +62,6 @@ module.exports = {
         allowArrowFunctions: true
       }
     ],
-    "function-paren-newline": [
-      "error",   "consistent"
-    ],
     indent: [
```

Comment:
Not sure how ESLint reconciles the same rule listed twice in the same file. You're now turning this rule off above but still have it here.

Reply:
Good catch, I missed this and will remove in favor of the `off` rule listed at the top